.dev/sass/layouts/_header.scss

```diff
@@ -69,6 +69,7 @@ body.no-max-width .site-header-wrapper {
 		position: relative;
 		clear: both;
 
+		h1,
```

Comment:
@fjarrett What about adding the new `.page-title` declaration alongside the original `h1` element - that way if the `h1` is filtered to another element it's still targeted by `.page-title`?